zowex/src/comm.rs

```diff
@@ -65,7 +60,10 @@ use crate::proc::*;
  *      A Result containing a stream upon success.
  *      This function exits the process upon error.
  */
-pub fn comm_establish_connection(njs_zowe_path: &str, daemon_socket: &str) -> io::Result<DaemonClient> {
+pub fn comm_establish_connection(
+    njs_zowe_path: &str,
+    daemon_socket: &str,
+) -> io::Result<DaemonClient> {
```

Comment:
Curious if we should have prettier implemented in the repo. 🤔

Reply:
I formatted w/ `rust-analyzer` here, but we could use the [Prettier plugin for Rust](https://github.com/jinxdash/prettier-plugin-rust) if we decide to go that route 😁